lib/image-type.ts

```diff
@@ -1,3 +1,4 @@
+import { normalize as normalizePath } from "path";
```

Comment:
we do we sometimes use `normalize` as is and sometimes rename is to `normalizePath`?

Reply:
I will unify that as use just `normalizePath` as it's more explicit! 👍